compiler/ballerina-lang/src/main/java/io/ballerina/projects/BuildOptions.java

```diff
@@ -234,6 +241,13 @@ public BuildOptionsBuilder setOffline(Boolean value) {
             return this;
         }
 
+        /**
+         * Set experimental compilation option.
+         *
+         * @return Build options builder
+         * @deprecated Since language no longer supports experimental features
+         */
+        @Deprecated(forRemoval = true)
         public BuildOptionsBuilder setExperimental(Boolean value) {
             compilationOptionsBuilder.setExperimental(value);
```

Comment:
Can't we remove experimental from CompilationOptions?

Reply:
`BuildOption` is using `CompilationsOptions.experimental(..)`
https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/io/ballerina/projects/BuildOptions.java#L67

Reply:
Removed via https://github.com/ballerina-platform/ballerina-lang/pull/35835/commits/4dc75b8e45e9295b1f58fa60fc559c1a05952acc